packages/integration/src/mocks/auth/getSettings.ts

```diff
@@ -33,7 +33,7 @@ export default function mockGetSettings(context: Context) {
                 last_name: 'Smith',
                 non_pep_declaration: 1,
                 phone: '+66111111111',
-                place_of_birth: null,
+                place_of_birth: 'Thailand',
```

Comment:
Very specific 😄

Reply:
Just to specify a place 😆 if not integration test will fail